### PR TITLE
docs: Rename registration.entity to registration.entity_id

### LINKED ISSUE #921: Update node operator docs with entity_id config
Once Oasis Core 24.2 becomes the recommended version for Mainnet, we should update the node operator documentation to use `registration.entity_id` in configuration instead of requiring the entire entity descriptor file.

### PR DESCRIPTION
**Pull request by amela**

### CODE REVIEW

File: docs/node/run-your-node/keymanager-node/README.md [hunk 1]
```diff
@@ -90,9 +90,12 @@ runtime:
...
   registration:
-    # In order for the node to register itself, the entity.json of the entity
-    # used to provision the node must be available on the node.
-    entity: /node/etc/entity.json
+    # In order for the node to register itself, the entity ID of the entity
+    # must be avaliable to the node.
+    entity_id: {{ node_entity_id }}
+    # Alternatively the entity.json of the entity used to provision the node
+    # must be available on the node:
+    # entity: /node/entity/entity.json
```
Comment (kostko) [inline docs/node/run-your-node/keymanager-node/README.md]: I would skip this alternative as we should move away from supporting it in the future.

File: docs/node/run-your-node/keymanager-node/README.md [hunk 2]
```diff
@@ -121,6 +124,7 @@ Before using this configuration you should collect the following information to
 * `{{ seed_node_address }}`: The seed node address in the form `ID@IP:port`.
   * You can find the current Oasis Seed Node address in the Network Parameters page ([Mainnet], [Testnet]).
 * `{{ keymanager_runtime_orc_path }}`: Path to the key manager [ParaTime bundle](../paratime-node.mdx#the-paratime-bundle) of the form `/node/runtimes/foo-paratime.orc`.
+* `{{ node_entity_id }}`: The node's entity ID from the `entity.json` file.
   * You can find the current Oasis-supported key manager ParaTime in the Network Parameters page ([Mainnet], [Testnet]).
```
Comment (matevz) [inline docs/node/run-your-node/keymanager-node/README.md]: Switch the lines.

```suggestion
* `{{ entity_id }}`: The node's entity ID from the `entity.json` file.
* `{{ keymanager_runtime_orc_path }}`: Path to the key manager [ParaTime bundle](../paratime-node.mdx#the-paratime-bundle) of the form `/node/runtimes/foo-paratime.orc`.
  * You can find the current Oasis-supported key manager ParaTime in the Network Parameters page ([Mainnet], [Testnet]).
```

File: docs/node/run-your-node/keymanager-node/README.md [hunk 3]
```diff
@@ -90,9 +90,12 @@ runtime:
 
 worker:
   registration:
-    # In order for the node to register itself, the entity.json of the entity
-    # used to provision the node must be available on the node.
-    entity: /node/etc/entity.json
+    # In order for the node to register itself, the entity ID of the entity
+    # must be avaliable to the node.
+    entity_id: {{ node_entity_id }}
```
Comment (kostko) [inline docs/node/run-your-node/keymanager-node/README.md]: Maybe call the variable just `{{ entity_id }}`?

File: docs/node/run-your-node/keymanager-node/README.md [hunk 4]
```diff
@@ -90,9 +90,12 @@ runtime:
 
 worker:
   registration:
-    # In order for the node to register itself, the entity.json of the entity
-    # used to provision the node must be available on the node.
-    entity: /node/etc/entity.json
+    # In order for the node to register itself, the entity ID of the entity
+    # must be avaliable to the node.
```
Comment (kostko) [inline docs/node/run-your-node/keymanager-node/README.md]:
```suggestion
    # In order for the node to register itself, the entity ID must be set.
```

File: docs/node/run-your-node/sentry-node.md [hunk 5]
```diff
@@ -182,6 +182,8 @@ Before using this configuration you should collect the following information to
 
   on the sentry node.
 
+* `{{ node_entity_id }}`: This is node's entity ID form `entity.json`.
```
Comment (kostko) [inline docs/node/run-your-node/sentry-node.md]:
```suggestion
* `{{ node_entity_id }}`: This is node's entity ID from `entity.json`.
```

File: docs/node/run-your-node/validator-node.mdx [hunk 6]
```diff
@@ -171,6 +166,8 @@ of that machine.
   You can find the current Oasis Seed Node address in the Network Parameters
   page ([Mainnet], [Testnet]).
 
+* `{{ node_entity_id }}`: The node's entity ID form `entity.json`.
```
Comment (kostko) [inline docs/node/run-your-node/validator-node.mdx]:
```suggestion
* `{{ node_entity_id }}`: The node's entity ID from `entity.json`.
```

**Review by kostko — COMMENTED** [review]
It doesn't say how to get the entity ID from the entity JSON. Is it possible to use the CLI for this purpose? I guess one could either `cat` the JSON file or use `oasis wallet show myentity` to get the `Public Key: [...]`.

File: docs/node/run-your-node/keymanager-node/README.md (hunk 2 above)
Comment (amela) [reply docs/node/run-your-node/keymanager-node/README.md]: Why? `keymanager_runtime_orc_path` is above `entity_id` in the code

Comment (matevz) [reply docs/node/run-your-node/keymanager-node/README.md]: The `You can find the current Oasis-supported key manager ParaTime...`  subbullet refers to the `keymanager_runtime_orc_path` setting.

File: docs/node/run-your-node/validator-node.mdx [hunk 7]
```diff
@@ -132,8 +132,17 @@ Create a JSON file containing the **public key** of your entity by executing
...
-`entity.json`.
+Now, we can move on to configuring our Oasis node with the information from
+freshly generated `entity.json`.
+
+:::info
+
+You can obtain your entity ID by running the command: `cat entity.json`, or
+alternatively, by using the Oasis CLI with the following command:
+`oasis wallet show my_entity`. If you use the Oasis CLI, your entity ID will be
+displayed in the `Public Key` field.
```
Comment (matevz) [inline docs/node/run-your-node/validator-node.mdx]:
````suggestion
Alternatively, if your entity account is imported into the [Oasis CLI] named `my_entity`
run the following command `oasis wallet show my_entity`. Your entity ID will be
displayed in the `Public Key` field.

[Oasis CLI]: ../../general/manage-tokens/cli/README.md
````

File: docs/node/run-your-node/validator-node.mdx [hunk 8]
```diff
@@ -132,8 +132,17 @@ Create a JSON file containing the **public key** of your entity by executing
...
 ```
 
-Now, we can move on to configuring our Oasis node with the freshly generated
-`entity.json`.
+Now, we can move on to configuring our Oasis node with the information from
+freshly generated `entity.json`.
+
+:::info
+
+You can obtain your entity ID by running the command: `cat entity.json`, or
```
Comment (matevz) [inline docs/node/run-your-node/validator-node.mdx]:
```suggestion
You can obtain your entity ID by running the `cat entity.json` command and
reading out the `id` field.
```

File: docs/node/run-your-node/validator-node.mdx [hunk 9]
```diff
@@ -132,20 +132,27 @@ Create a JSON file containing the **public key** of your entity by executing
...
 [`oasis wallet create`]: ../../general/manage-tokens/cli/wallet.md#create
 [`oasis wallet import`]: ../../general/manage-tokens/cli/wallet.md#import
 [`oasis wallet import-file`]: ../../general/manage-tokens/cli/wallet.md#import-file
 [`oasis account entity init`]: ../../general/manage-tokens/cli/account.md#entity-init
-
-### Copy the Public Entity File Descriptor to `server`
-
-The Oasis node needs access to the public key of your entity. Copy your
-`entity.json` to `/node/etc/entity.json` on the `server`.
+[Oasis CLI]: ../../general/manage-tokens/cli/README.md
```
Comment (matevz) [inline docs/node/run-your-node/validator-node.mdx]:
```suggestion
[`oasis wallet show`]: ../../general/manage-tokens/cli/wallet.md#show
```

File: docs/node/run-your-node/validator-node.mdx [hunk 10]
```diff
@@ -132,20 +132,27 @@ Create a JSON file containing the **public key** of your entity by executing
...
+Now, we can move on to configuring our Oasis node with the information from
+freshly generated `entity.json`.
+
+:::info
+
+You can obtain your entity ID by running the `cat entity.json` command and
+reading out the `id` field.
+
+Alternatively, if your entity account is imported into the [Oasis CLI] named
+`my_entity` run the following command `oasis wallet show my_entity`. Your
```
Comment (matevz) [inline docs/node/run-your-node/validator-node.mdx]:
```suggestion
Alternatively, if your entity account is imported into the Oasis CLI you can use
the [`oasis wallet show`] command. Your
```

**Review by matevz — APPROVED** [review]
One more suggestion. Otherwise looks good.

**Pull request merged**